src/app/components/filter/filter.component.html

```diff
@@ -56,7 +63,11 @@ <h2>By date:</h2>
                     </mat-form-field>
                 </form>
             </div>
-            <div id ="childComponent" class="col-sm-12 col-lg-{{isShow ? 12 : 9}}">
+            <div *ngIf="isLoading | async" class="overlay">
+                <mat-progress-spinner class="spinner" [color]="color" [mode]="mode" [value]="value">
```

Comment:
kijken of hier commentaar bij kan dat dan niet voor gebruiker zichtbaar zal zijn